ci/action.yaml

```
@@ -0,0 +1,42 @@
+name: entci
+description: continuous integration for ent projects
+inputs:
+  tidy:
+    description: run go mod tidy after generate
+    required: false
+    default: ""
+  working-directory:
+    default: "."
+    required: false
+    description: where to run commands
```

```suggestion
    description: where to run commands
```

Any reason the descriptions do not start with uppercase?

---

Nope